packages/table/src/lib/Table.stories.tsx

```diff
@@ -459,3 +463,85 @@ export const _PaginatedTable: StoryObj<typeof Table> = {
     );
   },
 };
+
+export const _ExpandableTable: StoryObj<typeof Table> = {
+  render: (args: TableProps) => {
+    return (
+      <TableContainer>
+        <Typography id="table-title" sx={visuallyHidden}>
+          Table
+        </Typography>
+        <Table aria-labelledby="table-title" {...args}>
+          <TableHead>
+            <TableRow>
+              <TableCell><div style={visuallyHidden}>Expand Row</div></TableCell>
```

Comment:
I'm not following how specifically we'd be building it into the component since it's the child and not the tablecell itself, but i'm also not sure it'd be needed. Realistically, instead of the column header being blank it'd more likely be a expand/collapse all button. However since UX didn't have specs for an expandable row (outside of the responsive table) this is just a slimmed down version of what DC had in their mockup so I skimped a bit.